layouts/partials/footer.html

```diff
@@ -65,7 +65,7 @@ <h2 class="h2">Stay up to date</h2>
       m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
   })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
 
-  ga('create', '{{ .Site.Params.googleAnalytics }}', 'auto');
+  ga('create', '{{ .Site.GoogleAnalytics }}', 'auto');
```

Comment:
...and this would then remain unchanged (which would make it look more consistent, too)